applications/serial_lte_modem/overlay-full_fota.conf

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# This overlay requires nRF9160 DK board revision 0.14.0 or later.
```

Comment:
This phrasing could be interpreted such that this overlay only works on nRF9160 DKs, which is not the case (e.g. nRF9161 DK).
I think it could be changed to state more precisely that it's only nRF9160 DK board revisions earlier than 0.14.0 that are not supported.

Reply:
Changed the phrasing a bit. Is it better now?

Reply:
Yes, almost perfect!